Umbraco-StarterKit/Creating-Content/Set-Up-Footer/index.md

```diff
@@ -0,0 +1,57 @@
+---
+versionFrom: 8.0.0
+---
+
+# Set-up your footer
+
+In this guide you will learn to set up a nice looking footer.
+
+1. Go to the backoffice
+2. Click you node
+3. In the workspace scroll to the bottom and find the section called footer
+4. In the field saying “Footer Columns” click “Add Content” this will give you 7 options of content
+5. Let’s start by adding Navigation
+    - Click Navigation in the list
+    - Set column width to 25% as 100% would be the entire width of the page.
+    - Choose a heading for your navigation list, let’s call it links for now.
+    - Next got to the field called “Links” and select “Add” then a window pops out on the right -hand side.
+    - Navigate to the part of the window that says “ Link to page” if this is a clean install you will have the options of Unicorn Hotel or Global Content, If you have created a page you can select that one, however, if that is not the case lets create a link to Unicorn Hotel, we do that by clicking the text, that will mark the field with a green checkmark, finalize by clicking submit.
+    - Next, we get the option to select children, which is another layer of links. we won't need this for now.
+    - The final button here is "Only Render Children" this would make the node only display the child links not the main links.
```

Comment:
```suggestion
    - The final button here is "Only Render Children" which would make the node only display links to the child pages, instead of the selected one
```